libnopegl/src/backends/gl/pipeline_gl.c

```diff
@@ -22,6 +22,7 @@
 
 #include <string.h>
 
+#include "bindgroup_gl.h"
```

Comment:
You're moving a lot of code out of `pipeline_gl.c` here; are all the headers still needed?

Same remark in `pipeline_vk.c`